src/image.rs

```diff
@@ -62,6 +62,12 @@ pub struct Image {
     pub width: u32,
     /// Height of the image.
     pub height: u32,
+}
+
+/// Parameters which specify how to render an image.
+#[derive(Copy, Clone, PartialEq, Debug)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct ImageRenderParams {
```

Comment:
Thoughts on `ImageSampler`? That name is inspired by the docs in `ImageQuality`

Reply:
I'm not tied to the `ImageSampler` name - I think that stronger names for this can exist. However, I also still think that it's a stronger name than `ImageRenderParams`. Addressing your claims in turn. 
- Just because you're not familiar with the term, doesn't make it not a commonly used computer graphics term. And as far as I understand it, it is a suitable description.
- A sampler is the instructions for doing sampling. See for example [`textureSample`](https://www.w3.org/TR/WGSL/#texturesample), which takes a texture and a sampler.

That being said, I don't think this is a blocker here - I'm willing to hold my nose for it.

Reply:
The [`wgpu::SamplerDescriptor`](https://docs.rs/wgpu/26.0.1/wgpu/type.SamplerDescriptor.html) is similar, though (almost) a superset of this.